datadog/resource_datadog_dashboard.go

```diff
@@ -2940,6 +2957,120 @@ func buildTerraformScatterplotRequest(datadogScatterplotRequest *datadog.Scatter
 	return &terraformRequest
 }
 
+//
+// Service Level Objective Widget Definition helpers
+//
+
+func getServiceLevelObjectiveDefinitionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"title": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"title_size": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"title_align": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+		"view_type": {
+			Type:     schema.TypeString,
+			Optional: true,
```

Comment:
`Required: true,`

Reply:
I have made anything that was marked as "omitempty" in https://github.com/zorkian/go-datadog-api/pull/277/files as Optional. Should I ignore that API and allow these to mismatch in what is required and what is optional? (This applies throught this PR).

Reply:
for the SLO widget yes, these are required. 🙏